src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
Avoid using static fields for the FigureSupplier, as static methods and fields are generally considered bad practice in an object-oriented approach.